Build error on Ubuntu 15.04

```
➜ make
ninja -C /home/veegee/pyston/build/Debug pyston copy_stdlib copy_libpyston sharedmods ext_pyston ext_cpython
ninja: Entering directory `/home/veegee/pyston/build/Debug'
[7/80] Building CXX object src/CMakeFiles/PYSTON_OBJECTS.dir/runtime/iterators.cpp.o
FAILED: CCACHE_CPP2=yes /usr/bin/ccache /usr/bin/clang++   -DDEFAULT_PYTHON_MAJOR_VERSION=2 -DDEFAULT_PYTHON_MICRO_VERSION=6 -DDEFAULT_PYTHON_MINOR_VERSION=7 -DLLVMREV=230300 -DNVA
LGRIND -DTHREADING_USE_GIL=1 -DTHREADING_USE_GRWL=0 -D__STDC_CONSTANT_MACROS -D__STDC_LIMIT_MACROS -Qunused-arguments -fcolor-diagnostics  -Qunused-arguments -fcolor-diagnostics -W
all -Wextra -Werror -Wreturn-type -Wno-sign-compare -Wno-unused -Wno-unused-parameter -fno-omit-frame-pointer -g -std=c++11 -fno-rtti -fexceptions -fvisibility-inlines-hidden -ffun
ction-sections -fdata-sections -Woverloaded-virtual -Wno-invalid-offsetof -Wcast-qual -Wno-sign-conversion -Wnon-virtual-dtor -Winit-self -Wmissing-include-dirs -Wstrict-overflow=5
 -Wpointer-arith -Wtype-limits -Wwrite-strings -Wempty-body -Waggregate-return -Wmissing-field-initializers -Wredundant-decls -Winline -Wint-to-pointer-cast -Wlong-long -Wvla -Wno-
attributes -g -Qunused-arguments -fcolor-diagnostics -Wimplicit-int -Wstrict-prototypes -Wold-style-definition -Wnested-externs -Wpointer-to-int-cast -Wno-mismatched-tags -Wno-exte
rn-c-compat   -g -DBINARY_SUFFIX= -DBINARY_STRIPPED_SUFFIX=_stripped -Isrc -I../../src -Ifrom_cpython/Include -I/home/veegee/pyston_deps/llvm-trunk/include -Illvm/include -I. -Ibui
ld_deps/libunwind/include -I../../build_deps/libpypa/src -I../../build_deps/lz4/lib -MMD -MT src/CMakeFiles/PYSTON_OBJECTS.dir/runtime/iterators.cpp.o -MF src/CMakeFiles/PYSTON_OBJ
ECTS.dir/runtime/iterators.cpp.o.d -o src/CMakeFiles/PYSTON_OBJECTS.dir/runtime/iterators.cpp.o -c ../../src/runtime/iterators.cpp
../../src/runtime/iterators.cpp:66:10: error: 'gc_visit' overrides a member function but is not marked 'override' [-Werror,-Winconsistent-missing-override]
    void gc_visit(GCVisitor* v) {
         ^
../../src/core/types.h:437:18: note: overridden virtual function is here
    virtual void gc_visit(GCVisitor* v) = 0;
                 ^
../../src/runtime/iterators.cpp:118:10: error: 'gc_visit' overrides a member function but is not marked 'override' [-Werror,-Winconsistent-missing-override]
    void gc_visit(GCVisitor* v) {
         ^
../../src/runtime/iterators.cpp:128:33: note: in instantiation of template class 'pyston::(anonymous namespace)::BoxIteratorIndex<pyston::BoxedList>' requested here
        BoxIterator begin = new BoxIteratorList((BoxedList*)container);
                                ^
../../src/core/types.h:437:18: note: overridden virtual function is here
    virtual void gc_visit(GCVisitor* v) = 0;
                 ^
../../src/runtime/iterators.cpp:118:10: error: 'gc_visit' overrides a member function but is not marked 'override' [-Werror,-Winconsistent-missing-override]
    void gc_visit(GCVisitor* v) {
         ^
../../src/runtime/iterators.cpp:133:33: note: in instantiation of template class 'pyston::(anonymous namespace)::BoxIteratorIndex<pyston::BoxedTuple>' requested here
        BoxIterator begin = new BoxIteratorTuple((BoxedTuple*)container);
                                ^
../../src/core/types.h:437:18: note: overridden virtual function is here
    virtual void gc_visit(GCVisitor* v) = 0;
                 ^
../../src/runtime/iterators.cpp:118:10: error: 'gc_visit' overrides a member function but is not marked 'override' [-Werror,-Winconsistent-missing-override]
    void gc_visit(GCVisitor* v) {
         ^
../../src/runtime/iterators.cpp:138:33: note: in instantiation of template class 'pyston::(anonymous namespace)::BoxIteratorIndex<pyston::BoxedString>' requested here
        BoxIterator begin = new BoxIteratorString((BoxedString*)container);
                                ^
../../src/core/types.h:437:18: note: overridden virtual function is here
    virtual void gc_visit(GCVisitor* v) = 0;
                 ^
4 errors generated.
[7/80] Building CXX object src/CMakeFiles/PYSTON_OBJECTS.dir/codegen/irgen/irgenerator.cpp.o
FAILED: CCACHE_CPP2=yes /usr/bin/ccache /usr/bin/clang++   -DDEFAULT_PYTHON_MAJOR_VERSION=2 -DDEFAULT_PYTHON_MICRO_VERSION=6 -DDEFAULT_PYTHON_MINOR_VERSION=7 -DLLVMREV=230300 -DNVALGRIND -DTHREADING_USE_GIL=1 -DTHREADING_USE_GRWL=0 -D__STDC_CONSTANT_MACROS -D__STDC_LIMIT_MACROS -Qunused-arguments -fcolor-diagnostics  -Qunused-arguments -fcolor-diagnostics -Wall -Wextra -Werror -Wreturn-type -Wno-sign-compare -Wno-unused -Wno-unused-parameter -fno-omit-frame-pointer -g -std=c++11 -fno-rtti -fexceptions -fvisibility-inlines-hidden -ffunction-sections -fdata-sections -Woverloaded-virtual -Wno-invalid-offsetof -Wcast-qual -Wno-sign-conversion -Wnon-virtual-dtor -Winit-self -Wmissing-include-dirs -Wstrict-overflow=5 -Wpointer-arith -Wtype-limits -Wwrite-strings -Wempty-body -Waggregate-return -Wmissing-field-initializers -Wredundant-decls -Winline -Wint-to-pointer-cast -Wlong-long -Wvla -Wno-attributes -g -Qunused-arguments -fcolor-diagnostics -Wimplicit-int -Wstrict-prototypes -Wold-style-definition -Wnested-externs -Wpointer-to-int-cast -Wno-mismatched-tags -Wno-extern-c-compat   -g -DBINARY_SUFFIX= -DBINARY_STRIPPED_SUFFIX=_stripped -Isrc -I../../src -Ifrom_cpython/Include -I/home/veegee/pyston_deps/llvm-trunk/include -Illvm/include -I. -Ibuild_deps/libunwind/include -I../../build_deps/libpypa/src -I../../build_deps/lz4/lib -MMD -MT src/CMakeFiles/PYSTON_OBJECTS.dir/codegen/irgen/irgenerator.cpp.o -MF src/CMakeFiles/PYSTON_OBJECTS.dir/codegen/irgen/irgenerator.cpp.o.d -o src/CMakeFiles/PYSTON_OBJECTS.dir/codegen/irgen/irgenerator.cpp.o -c ../../src/codegen/irgen/irgenerator.cpp
../../src/codegen/irgen/irgenerator.cpp:2846:23: error: 'getCAPIExcDest' overrides a member function but is not marked 'override' [-Werror,-Winconsistent-missing-override]
    llvm::BasicBlock* getCAPIExcDest(llvm::BasicBlock* from_block, llvm::BasicBlock* final_dest,
                      ^
../../src/codegen/irgen/irgenerator.h:163:31: note: overridden virtual function is here
    virtual llvm::BasicBlock* getCAPIExcDest(llvm::BasicBlock* from_block, llvm::BasicBlock* final_dest,
                              ^
../../src/codegen/irgen/irgenerator.cpp:2909:23: error: 'getCXXExcDest' overrides a member function but is not marked 'override' [-Werror,-Winconsistent-missing-override]
    llvm::BasicBlock* getCXXExcDest(llvm::BasicBlock* final_dest) {
                      ^
../../src/codegen/irgen/irgenerator.h:162:31: note: overridden virtual function is here
    virtual llvm::BasicBlock* getCXXExcDest(llvm::BasicBlock* final_dest) = 0;
                              ^
2 errors generated.
[7/80] Building CXX object src/CMakeFiles/PYSTON_OBJECTS.dir/codegen/pypa-parser.cpp.o
ninja: build stopped: subcommand failed.
Makefile:674: recipe for target 'pyston_dbg' failed
make: *** [pyston_dbg] Error 1
```

--- a/src/codegen/irgen/irgenerator.cpp
+++ b/src/codegen/irgen/irgenerator.cpp
@@ -2844,7 +2844,7 @@ public:
     // As a special-case, a NULL value for final_dest means that this helper block should
     // instead propagate the exception out of the function.
     llvm::BasicBlock* getCAPIExcDest(llvm::BasicBlock* from_block, llvm::BasicBlock* final_dest,
-                                     AST_stmt* current_stmt) {
+                                     AST_stmt* current_stmt) override {
         llvm::BasicBlock*& capi_exc_dest = capi_exc_dests[final_dest];
         llvm::PHINode*& phi_node = capi_phis[final_dest];
 
@@ -2906,7 +2906,7 @@ public:
         return capi_exc_dest;
     }
 
-    llvm::BasicBlock* getCXXExcDest(llvm::BasicBlock* final_dest) {
+    llvm::BasicBlock* getCXXExcDest(llvm::BasicBlock* final_dest) override {
         llvm::BasicBlock*& cxx_exc_dest = cxx_exc_dests[final_dest];
         if (cxx_exc_dest)
             return cxx_exc_dest;

--- a/src/runtime/iterators.cpp
+++ b/src/runtime/iterators.cpp
@@ -63,13 +63,14 @@ public:
         return &_end;
     }
 
-    void gc_visit(GCVisitor* v) {
+    void gc_visit(GCVisitor* v) override {
         if (iterator)
             v->visit(iterator);
         if (value)
             v->visit(value);
     }
 };
+
 
 template <typename T> class BoxIteratorIndex : public BoxIteratorImpl {
 private:
@@ -115,7 +116,7 @@ public:
         return &_end;
     }
 
-    void gc_visit(GCVisitor* v) {
+    void gc_visit(GCVisitor* v) override {
         if (obj)
             v->visit(obj);
     }

